dex/testing/loadbot/loadbot.go

```diff
@@ -1,6 +1,8 @@
 // This code is available on the terms of the project LICENSE.md file,
 // also available online at https://blueoakcouncil.org/license/1.0.0.
 
+//go:build lgpl
```

Comment:
I'm fine with loadbot just using eth without the tag.  It's not a package that someone might import, nor an app we'd want to distribute, just a dev tool, so as devs we're fine with this.

Reply:
I think that registering the asset `_ "decred.org/dcrdex/client/asset/eth"` forces us to build with `--lgpl` anyhow.